package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "bluebird": "3.5.4",
     "csv": "5.1.1",
+    "csv-parse": "4.4.1",
```

Comment:
huh? New dependency for parsing? Why not keep using `csv`? 
https://csv.js.org/parse/

Reply:
updated here b99edd9